src/LoginPage/LoginPage.js

```diff
@@ -30,17 +30,14 @@ class LoginPage extends Component {
 			})
 				.then(response => response.json())
 				.then((data) => {
-					this.setState({id: data.user.id, userName: data.user.name})
+					this.setState({id: data.user.id, userName: data.user.name}, () => {
+						this.props.changeUserId(this.state);
+					});
+					this.props.history.push(`/users/${this.state.id}`);
```

Comment:
Not sure exactly what this.props.history is doing, going to have to look over that